packages/devextreme-react/src/pivot-grid-field-chooser.ts

```diff
@@ -70,7 +70,6 @@ type IHeaderFilterProps = React.PropsWithChildren<{
   allowSelectAll?: boolean;
   height?: number;
   search?: HeaderFilterSearchConfig;
-  searchTimeout?: number;
```

Comment:
Why was the ```allowSearch``` option didn't delete here?

Reply:
because it wasn't deprecated it pivotgrid's fieldChooser